demo/index.html

```diff
@@ -128,5 +132,30 @@ <h3 id="no-icons">image-carousel demo with no navigation icons</h3>
         <div></div>
       </demo-snippet>
     </div>
+
+
+    <div class="vertical-section-container centered">
+      <h3>Basic image-carousel, used inside another polymer element</h3>
+      <demo-snippet>
+        <template>
+          <image-carousel-demo-app aria-labelledby="image-carousel inside another polymer element" images='[
+            "https://c2.staticflickr.com/2/1217/4607963354_e1a8fea210_z.jpg",
+            "https://cs1.livemaster.ru/storage/e6/45/00b119c13ed9f8b4dc6363a60d--materialy-dlya-tvorchestva-nabor-dlya-vyshivki-biserom-belyj-ti.jpg",
+            "https://tse4.mm.bing.net/th?id=ORT.TH_470633631&pid=1.12&eid=G.470633631"]'>
+          </image-carousel-demo-app>
+        </template>
+        <dom-bind>
+          <template>
+            <image-carousel-demo-app aria-labelledby="image-carousel inside another polymer element" images='[
+              "https://c2.staticflickr.com/2/1217/4607963354_e1a8fea210_z.jpg",
+              "https://cs1.livemaster.ru/storage/e6/45/00b119c13ed9f8b4dc6363a60d--materialy-dlya-tvorchestva-nabor-dlya-vyshivki-biserom-belyj-ti.jpg",
+              "https://tse4.mm.bing.net/th?id=ORT.TH_470633631&pid=1.12&eid=G.470633631"]'>
+            </image-carousel-demo-app>            
+          </template>
+        </dom-bind>
+        <div></div>
+      </demo-snippet>
+    </div>
+    
```

Comment:
Can you follow the pattern of the rest of the examples i.e. 3 new lines before the example and no new line between closing tag of `div` and `body`.

Reply:
done.